g/go-kmsg-parser/go-kmsg-parser_ubi_8.3.sh

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : go-kmsg-parser
+# Version               : 2.0.1,2.1.0
+# Source repo           : https://github.com/euank/go-kmsg-parser
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S<asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "go-kmsg-parser" ] ; then
+  rm -rf go-kmsg-parser
+fi
+
+# Dependency installation
+sudo dnf install -y git make golang
+# Download the repos
+git clone https://github.com/euank/go-kmsg-parser
+
+
+# Build and Test go-kmsg-parser
+cd go-kmsg-parser
+git checkout $VERSION
+ret=$?
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+
+#Build and test
+make test-deps
```

Comment:
Is it required and doing anything ? Rest all LGTM

Reply:
it does not do anything for latest version of the package ,but it requires for older version 2.0.1
pls refer https://github.com/euank/go-kmsg-parser/blob/v2.0.1/.travis.yml
the followins are installed as dep of the package

go get github.com/stretchr/testify/assert
go: downloading github.com/stretchr/testify v1.7.0
go: found github.com/stretchr/testify/assert in github.com/stretchr/testify v1.7.0
go: github.com/stretchr/testify upgrade => v1.7.0
go: downloading gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
Install dep success